examples/secp256k1-transfer/src/index.tsx

```diff
@@ -0,0 +1,173 @@
+import "bulma/css/bulma.css";
+import React, { useEffect, FC, ReactNode } from "react";
+import { useList, useSetState } from "react-use";
+import ReactDOM from "react-dom";
+import { nanoid } from "nanoid";
+import { BI } from "@ckb-lumos/lumos";
+import { fetchAddressBalance, generateAccountFromPrivateKey, transfer, Account, MIN_CELL_CAPACITY } from "./lib";
+
+type TransferTarget = {
+  capacity: BI;
+  address: string;
+  key: string;
+};
+
+const createTransferTarget = (): TransferTarget => ({ key: nanoid(), capacity: MIN_CELL_CAPACITY, address: "" });
+
+export function App() {
+  const [state, setState] = useSetState({
+    privateKey: "",
+    accountInfo: null as Account | null,
+    balance: BI.from(0),
+    txHash: "",
+    fee: BI.from(0),
+  });
+  const [transferTargets, transferTargetsActions] = useList<TransferTarget>([createTransferTarget()]);
+
+  // this method will be called when you click "Transfer" button
+  const doTransfer = () => {
+    if (!state.accountInfo) {
+      return;
+    }
+
+    transfer({ targets: transferTargets, address: state.accountInfo.address }, state.privateKey).then(
```

Comment:
The `transfer` function return `fee` field has a bit strange
I understand that if you want the user to confirm the fee before signing then you should separate the creation and signing steps

Reply:
But it is a example. Abstract is good for development, but not the best for here.
Tells user the transaction fee is from `sum(capacity of inputs) - sum(capacity of outputs)` is more important